EasyCommands.Tests/ScriptTests/FunctionalTests/Variables/VectorVariableTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace EasyCommands.Tests.ScriptTests {
+    [TestClass]
+    public class VectorVariableTests {
```

Comment:
Let's add more tests in the TernaryOperand tests testing ternary operations and vector variables combined.

Cases like ```a ? b:c:d : e``` and ```a ? b : c:d:e```

I think that'll help show why the other pull request is warranted.

We don't have tests here that actually exercise the VectorVariable check, that I can see.  We should add these to make sure the check works as expected.

Reply:
i have a test in a stash which i used for checking the parser steps which does cover a few more cases.
but most of the the operator behavior is covered by the old vector tests if we decide to remove primitive vectors.